nextcloud/content.md

```diff
@@ -79,14 +81,23 @@ If you want to use named volumes for all of these, it would look like this:
 
 ```console
 $ docker run -d \
-	-v nextcloud:/var/www/html \
-	-v apps:/var/www/html/custom_apps \
-	-v config:/var/www/html/config \
-	-v data:/var/www/html/data \
-	-v theme:/var/www/html/themes/<YOUR_CUSTOM_THEME> \
-	%%IMAGE%%
+-v nextcloud:/var/www/html \
+-v apps:/var/www/html/custom_apps \
+-v config:/var/www/html/config \
+-v data:/var/www/html/data \
+-v theme:/var/www/html/themes/<YOUR_CUSTOM_THEME> \
+%%IMAGE%%
 ```
 
+### Custom volumes
+
+If mounting additional volumes under `/var/www/html`, you should consider:
+
+-	Confirming that [upgrade.exclude](https://github.com/nextcloud/docker/blob/master/upgrade.exclude) contains the files and folders that should persist during installation and upgrades; or
+-	Mounting storage volumes to locations outside of `/var/www/html`.
+
+> [!WARNING] You should note that data inside the main folder (`/var/www/html`) will be overridden/removed during installation and upgrades, unless listed in [upgrade.exclude](https://github.com/nextcloud/docker/blob/master/upgrade.exclude). The additional volumes officially supported are already in that list, but custom volumes will need to be added by you. We suggest mounting custom storage volumes outside of `/var/www/html` and if possible read-only so that making this adjustment is unnecessary. If you must do so, however, you may build a custom image with a modified `/upgrade.exclude` file that incorporates your custom volume(s).
```

Comment:
`> [!WARNING] ...` is a github-specific markdownism that probably will not translate well to Docker Hub